packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -38,6 +47,93 @@ const getIPAddress = async (url: string): Promise<string> => {
   });
 };
 
+const checkUserIsAdmin = (
```

Comment:
Can we rename this to something a bit more descriptive - In my eyes checkUserISAdmin would just return a true or false but this is doing a bit more than just checking if a user is admin as it throws also

```suggestion
const checkUserIsAdminOrThrow = (
```